src/test/java/org/apache/datasketches/theta/UnionImplTest.java

```diff
@@ -279,7 +290,7 @@ public void printlnTest() {
    * @param o value to print
    */
   static void println(final Object o) {
-    //System.out.println(o.toString()); //disable here
+    System.out.println(o.toString()); //disable here
```

Comment:
Did you intend for this to be uncommented?

Reply:
thank you!!

Reply:
Fixed.